internal/config/config_test.go

```diff
@@ -185,28 +184,21 @@ func TestNewWithDefaultOptions(t *testing.T) {
 }
 
 func TestValidate(t *testing.T) {
-	const defaultErrCount = 21 // expect this number of error messages by default because user-specific values are not set and multiple providers are defined by default
-	const azErrCount = 9
-	const gcpErrCount = 6
-
```

Comment:
I don't think the tests will actually catch changes / errors now. The problem we worked around was that the `Default()` config itself was not a valid config. Any tests relativ to the config generated by `Default()` would also need to check for changes in error count.
Maybe we should refactor this test setup in general, but the proposed changes make them brittle. 
We should be able to count how many errors are contained in the new multi error, right?